lib/internal/process/per_thread.js

```diff
@@ -269,6 +275,49 @@ function wrapProcessMethods(binding) {
     return true;
   }
 
+  function execve(execPath, args, env) {
+    const { isMainThread } = require('internal/worker');
```

Comment:
Since this is an experimental feature, we should emit a warning on usage.

Reply:
Added, thanks!